.github/workflows/ci-pr-snapshots.yml

```
@@ -0,0 +1,24 @@
+name: CI - Snapshot Release on PRs
+
+on: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)